app/controllers/user.js

```diff
@@ -31,3 +31,16 @@ exports.signIn = (req, res, next) =>
       throw badRequestError('sign in error');
     })
     .catch(next);
+
+exports.getAllUsers = (req, res, next) => {
+  const limit = req.query.limit || 10;
+  const page = req.query.page || 1;
+  const offset = (page - 1) * limit;
+  const orderBy = req.query.orderBy || 'email';
```

Comment:
If you want this order, you need to specify how sort is performed (DESC or ASC), could be done like `email (for ASC order) and -email(for DESC order)`